include/common/arch/aarch64/asm/linkage.h

```diff
@@ -8,13 +8,13 @@
 
 #define GLOBAL(name) \
 	.globl name; \
-	name:
+name:
```

Comment:
Thanks for pointing this out, so there are at least to cases where label alignment looked better: in asm code and in macros. And putting `clang-format off` there everywhere would be a mess.

> don't like the idea to force this policy

I agree in part, meaning that I don't like forcing changes in already written code like in this PR and what we did before with clang-format patches.

But do we really want to force aligned labels everywhere instead?

Maybe it's time to change what `make indent` does? Maybe it should not handle all the code, so that we don't need to rewrite all existing code base on clang-format changes. But only handle identified number of commits?

Reply:
Please take a look on https://github.com/checkpoint-restore/criu/pull/2044